src/Microsoft.Identity.Client/ApiConfig/AbstractAcquireTokenParameterBuilder.cs

```diff
@@ -92,6 +92,20 @@ public T WithExtraQueryParameters(Dictionary<string, string> extraQueryParameter
             return (T)this;
         }
 
+
+        /// <summary>
+        /// Sets claims in the query. Use when the AAD admin has enabled conditional access. Acquiring the token normally will result in a
+        /// <see cref="MsalServiceException"/> with the <see cref="MsalServiceException.Claims"/> property set. Retry the 
+        /// token acquisition, and use this value in the <see cref="WithClaims(string)"/> method. See https://aka.ms/msal-exceptions for details
+        /// </summary>
+        /// <param name="claims">A string with one or multiple claims.</param>
+        /// <returns>The builder to chain .With methods</returns>
+        public T WithClaims(string claims)
```

Comment:
Do we want this to be a list (like we do with scopes) and then we can format it appropriately?

Reply:
Not sure, because users are supposed to get the value from MsalExceptions.Claims, which is a string. We could support both if needed.

Reply:
no this should be a string.
In the future we'd have another override (.WithClaimsRequest) which will take a complex object. But for the moment we are working on parity.